packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -1060,6 +1146,95 @@ capacity provider. The weight value is taken into consideration after the base v
   readonly weight?: number;
 }
 
+/**
+ * The details of the execute command configuration. For more information, see
+ * [ExecuteCommandConfiguration] https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-cluster-executecommandconfiguration.html
+ */
+export interface ExecuteCommandConfiguration {
+  /**
+   * The AWS Key Management Service key ID to encrypt the data between the local client and the container.
+   *
+   * @default - none
+   */
+  readonly kmsKey?: kms.IKey,
+
+  /**
+   * The log configuration for the results of the execute command actions. The logs can be sent to CloudWatch Logs or an Amazon S3 bucket.
+   *
+   * @default - No logs provided
+   */
+  readonly logConfiguration?: ExecuteCommandLogConfiguration,
+
+  /**
+   * The log setting to use for redirecting logs for the execute command results.
+   *
+   * @default - 'DEFAULT'
```

Comment:
Do we really set this to DEFAULT when creating the cluster construct?

Reply:
No, we don't really set it. Will update this.